spk/gerrit/Makefile

```diff
@@ -0,0 +1,54 @@
+SPK_NAME = gerrit
+SPK_VERS = 2.11.4
+SPK_REV = 1
+SPK_ICON = src/gerrit.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/busybox
+SPK_DEPENDS = "JavaManager:git"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Gerrit provides web based code review and repository management for the Git version control system.
+ADMIN_PROTOCOL = https
+ADMIN_PORT = 8091
```

Comment:
Can you move out of the 80* range, as per https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports#how-to-choose-a-port?